### DISCUSSION TITLE
Finding a minor planet

### DISCUSSION THREAD

**Discussion by jeffwynn** [Stellarium/stellarium#698] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Not certain yet. I'm a NOOB here. I cannot see an answer in the manual. 

**Describe the solution you'd like**
Include a particular minor planet on the sky chart.

**Describe alternatives you've considered**
I used to have TheSky planetarium software and could track this object, but it isn't supported anymore.

**Additional context**
Carolyn Shoemaker named a Phocaea asteroid after me: 9564 Jeffwynn  
I'd like to know where it is to try to see it with my 36 cm Dobsonian.

**Comment by gzotti** [comment, marked as answer]
The answer to your question is in section 12.9 of the User Guide. Alternatively, in appendix D.2.2.

**Comment by jeffwynn** [comment]
Wow - THANX!

What an extraordinary thing you and your colleagues are doing.

==Jeff



On Wed, Jun 19, 2019 at 1:58 PM Georg Zotti <notifications@github.com>
wrote:

> The answer to your question is in section 12.9 of the User Guide.
> Alternatively, in appendix D.2.2.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Stellarium/stellarium/issues/698?email_source=notifications&email_token=AMM2WDKHEGK5HR3CICXHWELP3KMWFA5CNFSM4HZNHI4KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODYDILKI#issuecomment-503743913>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMM2WDPNW3RXCCKT3OOX6TDP3KMWFANCNFSM4HZNHI4A>
> .
>


-- 
Jeff Wynn, PhD
Research Geophysicist
US Geological Survey
https://www.usgs.gov/staff-profiles/jeff-wynn
-- Wynn Sensei
sensei@vancouverjujitsu.org
